src/scripts/main.js

```diff
@@ -1,29 +1,58 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const validTypes = ['success', 'error', 'warning'];
+
+  if (!validTypes.includes(type)) {
+    throw new Error(
+      `Invalid notification type: "${type}". Expected one of ${validTypes.join(', ')}.`,
+    );
+  }
+
+  const div = document.createElement('div');
+
+  div.classList.add('notification', type);
+  div.style.top = `${posTop}px`;
+  div.style.right = `${posRight}px`;
+
+  const titles = document.createElement('h2');
+
+  titles.className = 'title';
+  titles.textContent = title;
+  div.appendChild(titles);
+
+  const descriptions = document.createElement('p');
+
+  descriptions.textContent = description;
+  div.appendChild(descriptions);
```

Comment:
The same applies to the 'descriptions' variable. It would be better to name it 'descriptionElement' or 'description'.